### DISCUSSION TITLE
Roll back of release 5.3.2.2

### DISCUSSION THREAD

**Discussion by VPKSoft** [desjarlais/Scintilla.NET#48] [Announcements]
Hi,
I had to roll back the changes from the v.5.3.2.2 as it broke the WinForms designer - sorry for the inconvenience. I released 5.3.2.3 to fix this.

**Comment by desjarlais** [comment]
This is about to be a year old issue, closing the discussion but if there is a need to re-visit the conversation, anyone else is free to re-open.
